### ISSUE TITLE
cron job to backup save files

### ISSUE THREAD

**Issue by GMBridge** [thijsvanloef/palworld-server-docker#39]
considering how unstable the game is, automatic backups every hour or so (keeping the last X backups) would be helpful if someone needs to roll back to recover after a bug or crash.

**Comment by Twinki14** [comment]
This might be better handled outside the docker image

**Comment by mpfiv** [comment]
also a cron job to periodically restart would be a nice addition if you are keeping backups x far back.

**Comment by thijsvanloef** [comment]
Open PR https://github.com/thijsvanloef/palworld-server-docker/pull/86 to create backups with a single command

**Comment by cinderisles** [comment]
I added a cronjob for the backup command added in #86 by adding `cron` to the dependencies installed in the `Dockerfile`

then added this to `scripts/init.sh`

```bash
echo "*/5 * * * * root /usr/local/bin/backup > /proc/1/fd/1 2>&1" > /etc/cron.d/backup
service cron start
```

this runs the backup command every 5 minutes

**Comment by llnut** [comment]
> I added a cronjob for the backup command added in #86 by adding `cron` to the dependencies installed in the `Dockerfile`
> 
> ```
> RUN apt-get update && apt-get install -y --no-install-recommends \
>     xdg-user-dirs=0.17-2 \
>     procps=2:3.3.17-5 \
>     wget=1.21-1+deb11u1 \
>     cron\
>     && apt-get clean \
>     && rm -rf /var/lib/apt/lists/*
> ```
> 
> then added this to `scripts/init.sh`
> 
> ```shell
> echo "*/5 * * * * root /usr/local/bin/backup > /proc/1/fd/1 2>&1" > /etc/cron.d/backup
> service cron start
> ```
> 
> this runs the backup command every 5 minutes
> 
> see [my fork](https://github.com/cinderisles/palworld-server-docker/commit/53fb698d6878187acfef25f76446227913771c01#diff-e28b09372ab7f9778c87c348a91532b491c793208b25f9f0f7b46b7154db45ff) (ignore the other file changes ._.)

I don't think automatic backup requires as much attention to the timing of the trigger as an automatic restart service does. If I were you, I would choose the following docker-compose configuration to implement the backup function, as it is obviously simpler:
```
backuper:
  image: docker:cli
  volumes: ["/var/run/docker.sock:/var/run/docker.sock"]
  command: ["/bin/sh", "-c", "while true; do sleep 900; docker exec palworld-server backup; done"]
  restart: unless-stopped
```
of course, this is just my own opinion.

**Comment by Dashboy1998** [comment]
Solved by #194.
